plugins/SimulationView/SimulationView.py

```diff
@@ -400,6 +402,9 @@ def getMinFeedrate(self) -> float:
     def getMaxFeedrate(self) -> float:
         return self._max_feedrate
 
+    def getSimulationTime(self) -> list:
+        return [length / feedrate for length, feedrate in zip(self._visible_lengths[self._current_layer_num], self._current_feedrates[self._current_layer_num])]
```

Comment:
See comment above about using Numpy instead of list comprehensions

Reply:
Keep in mind that feedrate is probably in a non standard unit `mm/min` instead of `mm/s` It might be that it already converted to seconds, but double check if you're a factor 60 of

Reply:
Also check if you're comparing the same arrays starting from the same offsets, since you calculate length using a sliding view

Reply:
yes I filter these arrays w.r.t. visible indexes

Reply:
I believe the feedate is already converted to seconds.  So the output here out of simulation time is in seconds

Reply:
457cecba5943c09b1b3b5b2a573a18c0915e0bc0